src/Sylius/Component/Promotion/Checker/Rule/CartQuantityRuleChecker.php

```diff
@@ -21,7 +21,7 @@
     '1.13',
     'The "%s" class is deprecated and will be removed in Sylius 2.0. Use "%s" instead.',
     CartQuantityRuleChecker::class,
-    \Sylius\Component\Core\Promotion\Checker\Rule\CartQuantityRuleChecker::class,
+    'Sylius\Component\Core\Promotion\Checker\Rule\CartQuantityRuleChecker',
```

Comment:
But why? If fqcn syntax doesn't work anymore for some reason, it should be possible to add `use CartQuantityRuleChecker as CoreCartQuantityRuleChecker`

Reply:
It's just to fix phparkitect error which detects this usage when using FQCN.